src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -0,0 +1,379 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import StylePropable from '../mixins/style-propable';
+import Transitions from '../styles/transitions';
+import DropDownArrow from '../svg-icons/navigation/arrow-drop-down';
+import Menu from '../menus/menu';
+import MenuItem from '../menus/menu-item';
+import DefaultRawTheme from '../styles/raw-themes/light-raw-theme';
+import ClearFix from '../clearfix';
+import ThemeManager from '../styles/theme-manager';
+import Popover from '../popover/popover';
+import PopoverAnimationFromTop from '../popover/popover-animation-from-top';
+import warning from 'warning';
+
+const DropDownMenu = React.createClass({
+
+  mixins: [StylePropable],
```

Comment:
I don't know how to use it yet. I'll try to look at your PRs and learn it good, Thanks for mentioning this :+1: